src/sections/Blog/Blog-grid/usedataList.js

```diff
@@ -25,6 +25,7 @@ const useDataList = (
     dataToSearch.sanitizer = new JsSearch.LowerCaseSanitizer();
     dataToSearch.searchIndex = new JsSearch.TfIdfSearchIndex(paramSearch);
     dataToSearch.addIndex(paramsIndex);
+    dataList[0].body && dataToSearch.addIndex("body");
```

Comment:
So I found out something about the list. The list items are a way to render fields that are nested inside each other and if you want to include independent fields, you won't add them into the list, you will have to add them as a separate argument and then do `dataToSearch.index(newlyAddedfield)` where the newlyaddedfield is the field you want to include in the search. I wanted to make the array in the paramIndex as an array of arrays and then use array index to iterate through them. But since I only wanted to index just the `body` I had to modify the `useDataList` to check if my data has a body and if it does, it should include it in its index to search. It's more robust. Infact, the modification even helps search on the author's name.